src/tab.group.ts

```diff
@@ -48,11 +51,10 @@ export default class TabGroup extends LitElement {
 
   override firstUpdated() {
     const selectedTab =
-      this.#tabElements?.find((tab) => tab.selected) ?? this.#tabElements?.[0];
+      this.#tabElements.find((tab) => tab.selected) ?? this.#tabElements[0];
 
     if (selectedTab) {
       this.#selectedTab = selectedTab;
-      this.#updateTabsPanelsAndTabBar();
```

Comment:
Doesn't seem useful here given it's called in `#onNavSlotChange()`, which is called after `firstUpdated()`.